packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
+  .reduceMotion(ReduceMotion.Never)
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
 
-<video
-  src="https://user-images.githubusercontent.com/48885911/134479266-5c7342f5-7453-4389-a6c5-5070e3673822.mov"
-  controls="controls"
-  muted="muted"
-  height="600"
+- `.duration(duration: number)` sets length of the animation (in milliseconds). Defaults to '300'.
+- `.delay(durationMs: number)` is the delay before the animation starts (in milliseconds). Defaults to `0`.
+- `.easingX(easing: EasingFunction)` defines easing animation for x-position.
+- `.easingY(easing: EasingFunction)` defines easing animation for y-position.
+- `.easingWidth(easing: EasingFunction)` defines easing animation for width.
+- `.easingHeight(easing: EasingFunction)` defines easing animation for height.
+- `.reduceMotion(reduceMotion: ReduceMotion)` determines how the animation responds to the device's reduced motion accessibility setting.
+- `.withCallback(callback: (finished: boolean) => void)` is the callback that will fire after the animation ends. Sets `finished` to `true` when animation ends without interruptions, and `false` otherwise.
+
+### Example
+
+<ExampleVideo
+  sources={{
+    android:
+      '/react-native-reanimated/recordings/layout_transitions/android_curved_transition.mov',
+    ios: '/react-native-reanimated/recordings/layout_transitions/ios_curved_transition.mov',
+  }}
 />
 
-### Entry/Exit Transition
+### Platform compatibility
 
-Entry/Exit transition, lets you specify different animations for exiting from the current position and different animations for entering the new position with new dimensions. You can use all available predefined entering/entering animation or create your own one. Its duration equals the duration sum of entering and exiting. Also, be aware that you cannot use spring animations as entering or exiting as they don't have a fixed duration.
+<div className="platform-compatibility">
 
-#### Modifiers
+| Android | iOS | Web |
+| ------- | --- | --- |
+| ✅      | ✅  | ❌  |
 
-- `delay` (in ms) default: 0
-- `withCallback` callback that will fire after the exit animation ends
-- `randomDelay` randomize delay of the animation between 0 and provided delay ( or 1000ms if delay not provided )
-- `entering` animation that will be used for component entering ( default: `FadeIn`)
-- `exiting` animation that will be used for component exiting ( default: `FadeOut`)
-- `reduceMotion` determines how the animation responds to the device's reduced motion accessibility setting.
+</div>
 
-#### Combine Transition
+## Entry/Exit Transition
 
-To make usage of that transition simplier we have prepared function `combineTransition` that will make your code look cleaner and shorter.
+Entry/Exit transition lets you choose layout animations for entering and exiting. Moreover, instead of picking currently available animations, you are welcome to create your own one.
```

Comment:
It suggests a bit that it sets animations for mounting and unmounting instead of combining both animations into a single one and applying them on layout changes.

Reply:
I changed it so it includes the `combining` part, let me know what you think C:

Reply:
Entry/Exit transition lets you choose layout animations for entering and exiting. It combines both of them into a single one and applies to the layout. Moreover, instead of picking currently available animations, you are welcome to create your own one.